indexer/src/main/java/bio/terra/tanagra/indexing/job/dataflow/beam/BigQueryBeamUtils.java

```diff
@@ -65,6 +69,22 @@ public static PCollection<KV<Long, Long>> readTwoFieldRowsFromBQ(
                 }));
   }
 
+  /**
+   * Build a query to select all ancestor-descendant pairs from the ancestor-descendant table, and
+   * the pipeline step to read the results.
+   */
+  public static PCollection<KV<Long, Long>> readAncestorDescendentRelationshipsFromBQ(
+      Pipeline pipeline, ITHierarchyAncestorDescendant ancestorDescendantTable) {
+    String ancestorDescendantSql =
+        "SELECT * FROM " + ancestorDescendantTable.getTablePointer().render();
+    LOGGER.info("ancestor-descendant query: {}", ancestorDescendantSql);
+    return BigQueryBeamUtils.readTwoFieldRowsFromBQ(
+        pipeline,
+        ancestorDescendantSql,
+        ITHierarchyAncestorDescendant.Column.DESCENDANT.getSchema().getColumnName(),
```

Comment:
function and log says ancestor-descendant
are the two column names interchanged?